utils/mkhtml.py

```diff
@@ -164,23 +164,22 @@ def download_git_commit(url, response_format, *args, **kwargs):
             desc = HTTP_STATUS_CODES[index].description
             gs.fatal(
                 _(
-                    "Download commit from <{url}>, return status code "
-                    "{code}, {desc}".format(
-                        url=url,
-                        code=response.code,
-                        desc=desc,
-                    ),
+                    "Download commit from <{url}>, return status code {code}, {desc}"
+                ).format(
+                    url=url,
+                    code=response.code,
+                    desc=desc,
                 ),
             )
```

Comment:
Off topic: Should this be even translatable? It is compilation. Perhaps called though _g.extension_? But still, is this something people should spend their time with translating to other languages?